packages/meditrak-app/app/navigation/Navigator.js

```diff
@@ -40,14 +40,17 @@ const routes = {
   [CHANGE_PASSWORD_SCREEN]: { screen: ChangePasswordContainer },
   [LOGIN_SCREEN]: { screen: LoginContainer, navigationOptions: () => ({ headerShown: false }) },
   [REQUEST_COUNTRY_ACCESS_SCREEN]: { screen: RequestCountryAccessContainer },
-  [WELCOME_SCREEN]: { screen: WelcomeContainer, navigationOptions: () => ({ headerShown: false }) },
+  [WELCOME_SCREEN]: {
+    screen: WelcomeContainer,
+    navigationOptions: () => ({ headerShown: false, animationEnabled: false }),
+  },
   [CREATE_ACCOUNT_SCREEN]: {
     screen: CreateUserContainer,
     navigationOptions: () => ({ headerRight: () => null }),
   },
   [HOME_SCREEN]: {
     screen: HomeScreenContainer,
-    navigationOptions: () => ({ headerLeft: () => null }),
+    navigationOptions: () => ({ headerLeft: () => null, animationEnabled: false }),
```

Comment:
I couldn't figure out how in old version of meditrak-app, it does not have any transition animation for these screens... So for our upgrade, I just disable the animation in Welcome/Home screens so that they're just quick-replace instead of default horizontal slide from right.

I've also checked navigating to other screens and compare the transitions with our production app and they seem consistent (at least for me)